package.json

```diff
@@ -128,6 +128,7 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-dropzone": "^14.2.3",
+    "react-icons": "^4.12.0",
```

Comment:
What was this added for? Is there an icon in this package that we don't get in the regular material UI icons pack that we're using elsewhere?

Reply:
Wasn't necessary